src/components/PageTitle.tsx

```diff
@@ -29,6 +31,7 @@ export const PageTitle: React.FC<Props> = ({
           <BadgeWrapper>{badge}</BadgeWrapper>
         </span>
       </Subtitle>
+      {buttons ?? <></>}
```

Comment:
return value should be null instead of fragment so that it literally returns nothing